version.tf

```diff
@@ -1,18 +1,14 @@
 terraform {
-  # require 1.9 or later to make use of cross-object referencing for input variable validations
-  #   more info: https://www.hashicorp.com/blog/terraform-1-9-enhances-input-variable-validations
-  required_version = ">= 1.9.0"
-
-  #
-  # Developer tips:
-  #   - If your module requires any terraform providers, add them the "required_providers" section below.
-  #   - Each required provider's version should be a flexible range to future proof the module's usage with upcoming minor and patch versions.
-  #
-
+  required_version = ">= 1.0.0"
   required_providers {
+    # Use "greater than or equal to" range in modules
     ibm = {
-      source  = "IBM-Cloud/ibm"
-      version = ">= 1.71.2, < 2.0.0"
+      source  = "ibm-cloud/ibm"
+      version = ">= 1.70.0, < 2.0.0"
```

Comment:
suggest to update to `>=1.76.1, <2.0.0` to pick up latest fix for auth policy updates